lib/google_product_change.dart

```diff
@@ -0,0 +1,51 @@
+class GoogleProductChangeInfo {
```

Comment:
Should this be moved to the `models` folder? I guess it doesn't use `freezed` but it's still a model I would say.

Reply:
Yeah, I was actually torn about this 😅 I'll move them into `models` though! It looks better in there 🤷‍♂️